templates/components/itilobject/actors/field.html.twig

```diff
@@ -222,7 +222,7 @@
                   action: 'getActors',
                   actortype: actorytype,
                   users_right: '{{ users_right ?? 'all' }}',
-                  entity_restrict: {{ entities_id }},
+                  entity_restrict: (actors.requester.length == 0) ? -1 : {{ entities_id }},
```

Comment:
I may be wrong, but I think this `-1` value should only be used for new tickets.

Reply:
you're right